### PR TITLE
georgia 6

### PR DESCRIPTION
**Pull request by georgiacarr1999**

### CODE REVIEW

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java [hunk 1]
```diff
@@ -1,169 +1,161 @@
...
+    private int amountOfPlayers = players.size();
+    private int currentPlayer = 0;
+    private boolean isGettingOutOfPenaltyBox;
+    private static final int maxAmountOfQuestions = 50;
+    private static final int maximumPlaceOnTheBoard = 11;
+    private static final int startingPlaceOnTheBoard = 0;
+    private static final int startingAmountOfCoins = 0;
+
+    public Game() {
+        for (int index = 0; index < maxAmountOfQuestions; index++) {
```
Comment (HannahHuggins) [inline exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java]: Does this class include all of the categories?

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java [hunk 2]
```diff
@@ -1,169 +1,161 @@
...
+        if (currentQuestionCategory().equals("Science"))
+            System.out.println(scienceQuestions.removeFirst());
+        if (currentQuestionCategory().equals("Sports"))
+            System.out.println(sportsQuestions.removeFirst());
+        if (currentQuestionCategory().equals("Rock"))
+            System.out.println(rockQuestions.removeFirst());
+    }
+
+
+    private String currentQuestionCategory() {
```
Comment (HannahHuggins) [inline exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java]: In this method there is an inconsistent way of finding out where the player is on the board, and getting the category; do you mean to define another Boolean?

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Category.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+class Category {
+
+    private List<String> questions = new ArrayList<>();
+
+
+    public Category(String name) {
+
+        for (int index = 0; index < 50; index++) {
+            questions.add(("Pop Question " + index));
```
[the remark this answers is not in the document]
Comment (georgiacarr1999) [reply exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Category.java]: i was in the process of moving each one over but ran out of time :)

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java (hunk 1 above)
Comment (georgiacarr1999) [reply exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java]: yeah, was going to add each category later

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java [hunk 4]
```diff
@@ -130,6 +126,13 @@ public boolean questionWasAnsweredCorrectly() {
         }
     }
 
+    private void loopToSeeWhosNext() {
```
Comment (HannahHuggins) [inline exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java]: This makes the code look so much easier to read :)

File: exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java (hunk 2 above)
Comment (georgiacarr1999) [reply exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java]: tbh im not too sure :| haha

**Pull request closed without merging**